### ISSUE TITLE
Problems displaying on Paquet

### ISSUE THREAD

**Issue by m4rocks** [toniengelhardt/m3tadata#89]
Due to some legal issues, I can't store app icons and screenshots anymore without the developer's specific consent. I am still working on that. Until then, I am displaying the icons and other resources directly to the user.

Unfortunately your app has some cross-origin restrictions and the icon cannot be displayed anywhere else. Could you please fix that and get back to me, so I can put Merklin back on Paquet?

**Comment by toniengelhardt** [comment]
Hi @notangelmario,

What URL are you trying to access? When I'm opening the icons from `/manifest.json` in a browser, e.g. https://app.merklin.xyz/_nuxt/icons/512x512.0f659041.png, it seems to work.

You have my consent btw to use all the icons and images.

**Comment by notangelmario** [comment]
Yes it does work when openning them in the browser, but not when displayed in another origin. (eg. localhost:3000, paquet.shop)

**Comment by toniengelhardt** [comment]
Can you try again, just removed the [nuxt-security](https://nuxt-security.vercel.app) module, maybe that was the problem.

**Comment by notangelmario** [comment]
It's fixed! But I don't recommend removing the `nuxt-security` module all together. Try configuring the only the CORS handler as specified [here](https://nuxt-security.vercel.app/middlewares/cors-handler).

**Comment by toniengelhardt** [comment]
Perfect, will do that, which routes (images) do you need access to?

**Comment by notangelmario** [comment]
I noticed Paquet grabs the 192x192 image. If you would like to save bandwidth, I would recommend adding a 96x96 image because Paquet doesn't need more than that. It's up to you.

**Comment by toniengelhardt** [comment]
Ok perfect, will adjust.
